src/main/java/adris/altoclef/Settings.java

```diff
@@ -15,72 +16,64 @@
 import java.util.ArrayList;
 import java.util.List;
 
-@SuppressWarnings("ALL")
-public class Settings {
+
+@SuppressWarnings({ "FieldMayBeFinal", "FieldCanBeLocal", "MismatchedReadAndWriteOfArray", "DanglingJavadoc" })
+public class Settings { // TODO: 2021-05-22 move to json config please
```

Comment:
Hmm how would these settings be accessed then? There needs to be some kind of a serializable class that handles this, and these settings ARE saved+loaded from a JSON.

(the settings file is [used everywhere](https://github.com/adrisj7-AltoClef/altoclef/search?q=getModSettings%28%29), there has to be a clean way to grab these user settings)

Reply:
Ah, GSON.

check out my good friend, [Jackson](https://github.com/FasterXML/jackson), would you?